Content.Server/Power/EntitySystems/PowerNetSystem.cs

```diff
@@ -86,6 +89,11 @@ private void ApcPowerReceiverRemove(EntityUid uid, ApcPowerReceiverComponent com
             component.NetworkLoad.Paused = false;
         }
 
+        private void ApcPowerReceiverPowerChanged(Entity<AppearanceComponent> ent, ref PowerChangedEvent args)
+        {
+            _appearance.SetData(ent.Owner, PowerDeviceVisuals.Powered, args.Powered, ent.Comp);
```

Comment:
Because its the same system that raises event, it's probably better for performance to just have it use an entity query to get & update the appearance component directly when raising the event instead of adding an event handler

Reply:
Alright. Changed it back to a query.